### PR TITLE
nfdump: 1.6.16 -> 1.6.17

### PR DESCRIPTION
**Pull request by tobim**
###### Things done
Also enabled the build of a few additional tools.
Closure size increases from 33.8M to 35.0M.

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/networking/nfdump/default.nix [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
+    sha256 = "1z8zpvd9jfi2raafcbkykw55y0hd4mp74jvna19h3k0g86mqkxya";
   };
 
-  nativeBuildInputs = [yacc flex];
-  buildInputs = [bzip2];
+  nativeBuildInputs = [ autoconf automake flex libtool pkg-config yacc ];
+  buildInputs = [ bzip2 libpcap ];
+
+  preConfigure = ''
+    patchShebangs autogen.sh
```
Comment (markuskowa) [inline pkgs/tools/networking/nfdump/default.nix]: That should not be necessary.

File: pkgs/tools/networking/nfdump/default.nix [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+    LIBTOOLIZE=libtoolize ./autogen.sh
```
Comment (markuskowa) [inline pkgs/tools/networking/nfdump/default.nix]: This should be enough:
```suggestion
    ./autogen.sh
```

**Review by markuskowa — CHANGES_REQUESTED** [review]

**Comment by markuskowa** [comment]
Changelog: https://github.com/phaag/nfdump/releases

File: pkgs/tools/networking/nfdump/default.nix (hunk 2 above)
Comment (tobim) [reply pkgs/tools/networking/nfdump/default.nix]: This is just a workaround for darwin. See https://github.com/phaag/nfdump/blob/master/autogen.sh#L5-L7. I can add a condition around it if you wish.

Comment (markuskowa) [reply pkgs/tools/networking/nfdump/default.nix]: I don't think we need a condition in that case but you could add the comment here stating why it is necessary.

**Review by markuskowa — APPROVED** [review]

**Pull request merged**